Composer/.eslintrc.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
     '@typescript-eslint/no-empty-function': 'off',
     '@typescript-eslint/no-unnecessary-type-assertion': 'off',
     '@typescript-eslint/no-use-before-define': 'warn',
+    '@typescript-eslint/prefer-optional-chain': 'error',
```

Comment:
Seems odd for a 'prefer' to cause an error.  Should we start with warn?

Reply:
I "prefer" erro :), save will autofix it so it should be fine.